.github/workflows/build-and-deploy-preprod.yml

```diff
@@ -54,10 +54,10 @@ jobs:
           credentials_json: ${{ secrets.SALSA_CREDENTIALS }}
 
       - name: Provenance, upload and sign attestation
-        uses: nais/salsa@v0.7
+        uses: nais/salsa@v0.6
```

Comment:
Måtte gå ned en versjon for å få det til å funke. Virker som det er det de fleste i nav bruker: https://github.com/search?q=org%3Anavikt+nais%2Fsalsa%40v&type=code